### PR TITLE
New VIB workflow 'first cut'

### PR DESCRIPTION
**Pull request by randradas**
**Description of the change**

This change introduces a couple of workflows that run a couple of VIB *first cut* pipelines.

**Benefits**

Automatic verification with VIB

### CODE REVIEW

File: .github/workflows/vib.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+    branches:
+      - stable
+    types:
+      - assigned
```
Comment (paleloser) [inline .github/workflows/vib.yaml:7]: Why would you want to re-run the workflow if the PR is assigned? That even't doesn't mean code changes, does it?

File: .vib/vib-platform-verify.json [hunk 2]
```diff
@@ -3,39 +3,33 @@
...
         }
       ]
     },
     "verify": {
       "context": {
         "resources": {
-          "url": "https://github.com/CartoDB/carto3-helm/tarball/bc22e8f8ece48277a6681ddb2dde37ca8b536ee8",
-          "path": "/tests"
+          "url": "{SHA_ARCHIVE}",
+          "path": "/chart"
```
Comment (paleloser) [inline .vib/vib-platform-verify.json:20]: I'd switch this to point at the base test directory, which it looks like it's going to be `/.vib/carto/`. Then, in the actions, for example `goss`, you can use the relative path `goss/goss-accounts-www`.

File: .github/workflows/vib.yaml (hunk 1 above)
Comment (randradas) [reply .github/workflows/vib.yaml:7]: I completely agree, I will remove it.

File: .github/workflows/vib.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+    branches:
+      - stable
+    types:
+      - opened
+      - synchronize
+      - reopened
```
Comment (paleloser) [inline .github/workflows/vib.yaml:9]: I **think** you don't need these anymore, they are the default types for [`pull_request`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

**Review by paleloser — APPROVED** [review]

**Pull request merged**